cli/cmd/testdata/inject_emojivoto_deployment_tls.golden.yml

```diff
@@ -126,14 +107,5 @@ spec:
           runAsNonRoot: false
           runAsUser: 0
         terminationMessagePolicy: FallbackToLogsOnError
-      volumes:
-      - configMap:
-          name: linkerd-ca-bundle
-          optional: true
-        name: linkerd-trust-anchors
```

Comment:
i still see references to `linkerd-trust-anchors`, should these go away completely? same question for `linkerd-trust-anchors`, `linkerd-secrets`, `/var/linkerd-io/identity`, and `/var/linkerd-io/trust-anchors`